Lib/unittest/test/test_case.py

```diff
@@ -596,6 +596,48 @@ def testShortDescriptionWithMultiLineDocstring(self):
                  'Tests shortDescription() for a method with a longer '
                  'docstring.')
 
+    @unittest.skipIf(
+        sys.flags.optimize >= 2,
+        "Docstrings are omitted with -O2 and above")
+    def testShortDescriptionWithSurroundingSpaceOneLineDocstring(self):
```

Comment:
I mean that adding spaces at the start and end of the docstring of testShortDescriptionWithOneLineDocstring and adding a newline and indentation at the start of the docstring of testShortDescriptionWithMultiLineDocstring will cover cases tested by new testes. No new tests will be needed.